dataStores/chat/ChatController.js

```diff
@@ -0,0 +1,32 @@
+const storage = require('../dataStores/Storage');
+
+class ChatController {
+
+	getMessages(req, res) {
+		return res.status(200).json({messages: storage.messages});
```

Comment:
200 to chyba standard, Ja bym usunął ale to niech jeszcze ktoś się wypowie

Reply:
Mi jakoś podawanie explicite statusu zawsze jest ok